doc/local-dev.md

```diff
@@ -0,0 +1,99 @@
+#### Goal
+The following steps are going to set up a dev environment with minikube. This is meant to allow you to test your changes to draughtsman before they are committed or merged.
+
+#### Preparation
```

Comment:
I would break this up into:
- setting up minikube
- setting up helm
- configuring draughtsman